cro-clib/src/address.rs

```diff
@@ -0,0 +1,77 @@
+use crate::types::{CroAddressPtr, CroResult};
+use std::ffi::CString;
+use std::ptr;
+/// print address information
+#[no_mangle]
+/// # Safety
+pub unsafe extern "C" fn cro_print_address(address_ptr: CroAddressPtr) -> CroResult {
```

Comment:
More flexible to return a formatted string then call print in rust.

Reply:
i agree, i'll change it